pkg/chunk/elastic/es_index_client.go

```diff
@@ -276,14 +279,25 @@ func NewESIndexClient(cfg Config) (chunk.IndexClient, error) {
 }
 
 func newES(cfg Config) (*elastic.Client, error) {
+	//fix x509: certificate signed by unknown authority
+	tr := &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
```

Comment:
in my opinion this is not correct way to handle "certificates". If you use insecure always it does not identify the elasticsearch server at all. If people want use insecure it should be startup flag or configuration option somewhere, but default value insecure is wrong

Reply:
make sense. I will fix it.

Reply:
done, plz review again.